lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<p>A TODO: for another time maybe.</p><p> </p><p>We could consider a note here commenting on the shape of the distribution and how it might differ from a commercial (fast) gamma calculation tool, where certain optimisations are made to obtain an accurate pass rate quickly at the expense of computing erroneously large gamma index values, which is evident in the shape of the gamma histogram.</p><p> </p><p>Probably something more for Simon or I later. We could note it in a Github issue maybe.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>